setup.cfg

```
@@ -2,41 +2,46 @@
 name = pysatSpaceWeather
 version = file: pysatSpaceWeather/version.txt
 url = https://github.com/pysat/pysatSpaceWeather
-author = Jeff Klenzing
-author_email = jeffrey.klenzing@nasa.gov
+author = Angeline Burrell
+author_email = pysat.developers@gmail.com
 description = 'pysat support for Space Weather Indices'
 keywords =
-  pysat
-  ionosphere
+    pysat
+    ionosphere
+    space weather
+    forecasting
+    indexes
 classifiers =
-  Development Status :: 3 - Alpha
-  Topic :: Scientific/Engineering :: Physics
-  Topic :: Scientific/Engineering :: Atmospheric Science
-  Intended Audience :: Science/Research
-  License :: OSI Approved :: BSD License
-  Natural Language :: English
-  Programming Language :: Python :: 3.5
-  Programming Language :: Python :: 3.6
-  Programming Language :: Python :: 3.7
-  Operating System :: MacOS :: MacOS X
-  Operating System :: POSIX :: Linux
+    Development Status :: 3 - Alpha
+    Topic :: Scientific/Engineering :: Physics
+    Topic :: Scientific/Engineering :: Atmospheric Science
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Natural Language :: English
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
```

```suggestion
    Programming Language :: Python :: 3.6
    Programming Language :: Python :: 3.7
    Programming Language :: Python :: 3.8
```
Should match supported tests on travis